### PR TITLE
Added a few changes from MaskedRetriever and me

### PR DESCRIPTION
**Pull request by steveway**
This adds support for a few things:
-Export of Imagesequences
-Mouths can now be any (not everything tested) by wx supported image type instead of only jpegs
-A converted example mouth as a .png with transparency is included.

### CODE REVIEW

**Comment by morevnaproject** [comment]
Looks cool! I need some time to test those change. Thank you! ^__^

File: setup_ubuntu.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+apt-get install python-pyaudio python-wxGlade
```
Comment (morevnaproject) [inline setup_ubuntu.sh]: It's enough to have python-wxgtk2.8 instead of python-wxGlade.
See https://morevnaproject.org/papagayo/ ^__^

Comment (steveway) [inline setup_ubuntu.sh]: Yeah, haven't touched that part. That file is from MaskedRetriever. Maybe he wanted to try around a bit with wxGlade. I would say this is more something that would fit into the Readme.

**Comment by morevnaproject** [comment]
@steveway I have tested the latest commit and it gives me error when I choose "Images" output type and click "Export voice...":

```
Traceback (most recent call last):
   File "/home/konstantin/Seafile.studio/workfiles/papagayo/LipsyncFrame.py", line 605, in OnVoiceExport
     self.doc.currentVoice.ExportImages(dlg.GetPaths()[0],self.mouthChoice.GetStringSelection())
   File "/home/konstantin/Seafile.studio/workfiles/papagayo/LipsyncDoc.py", line 377, in ExportImages
     for file in os.listdir(os.path.join(os.path.dirname(os.path.abspath(__file__)), "rsrc/mouths/")+currmouth):
 NameError: global name 'currmouth' is not defined
```

**Comment by steveway** [comment]
You are right, there was a typo in there.
Fixed this in the newest version.

**Comment by morevnaproject** [comment]
Merged now. Thank you! It turns to be a cool feature! ^__^

**Comment by steveway** [comment]
Yeah, I still have a few ideas for that. I'll add one as an Issue here.

**Pull request merged**